src/Questions.jsx

```diff
@@ -0,0 +1,140 @@
+import React, { useEffect, useState } from "react";
+const Question = (props) => {
+    const allAnswer = [
+        ...props.value.incorrect_answers,
+        props.value.correct_answer,
+    ];
+    const [selected, setSelected] = useState(null);
+    let styleQue = props.checkAns
+        ? selected === props.value.correct_answer
+            ? "bg-green-900  text-white border-none"
+            : "bg-red-900  text-white border-none opacity-50"
+        : "bg-violet-200 text-slate-700 border-none";
+
+    return (
+        <div className=" border-b py-2">
+            <h2
+                className="text-xl font-semibold text-slate-700"
+                dangerouslySetInnerHTML={{ __html: props.value.question }}
+            ></h2>
+            <div className="flex gap-2 lg:flex-nowrap flex-wrap my-3">
+                {allAnswer.sort().map((v, k) => {
```

Comment:
I suggest you use more  descriptive parameters, like , instead of passing v and k in the map  function you can use something meaning so that it is easy to ready your code.